db.js

```diff
@@ -0,0 +1,66 @@
+import mongoose from 'mongoose'
+import dotenv from 'dotenv'
+
+dotenv.config()
+
+mongoose.set('strictQuery', true)
+
+async function dbClose() {
+    await mongoose.connection.close()
+    console.log("Database disconnected!")
+}
+
+// Connect to MongoDB via Mongoose
+try {
+    const m = await mongoose.connect(process.env.ATLAS_DB_URL)
+    console.log(m.connection.readyState === 1 ? 'Mongoose connected!' : 'Mongoose failed to connect')
+}
+catch (err) {
+    console.log(err)
+}
+
+// Define Book schema
+const bookSchema = new mongoose.Schema({
```

Comment:
add the below?
condition
location
language
genre
description

Reply:
amending